### PR TITLE
Features/xo lint

### PR DESCRIPTION
**Pull request by Mikanebu**
* installed xo as dev dependancy
* setup configs in package.json
* run and fixed all errors (xo --fix to auto-fix)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -31,29 +26,28 @@ class CoreTools {
...
         content = fs.readFileSync(path_)
       } catch (err) {
         error(err.message)
-        process.exit(1)
       }
-      // Get JS object from file content
+      //  Get JS object from file content
       const descriptor = JSON.parse(content)
-      //descriptor.profile = 'https://schemas.frictionlessdata.io/data-package.json'
+      //  Descriptor.profile = 'https://schemas.frictionlessdata.io/data-package.json'
```
Comment (anuveyatsu) [inline index.js]: We don't need this anymore - just delete it

File: index.js [hunk 2]
```diff
@@ -31,29 +26,28 @@ class CoreTools {
...
       console.log(`checking following packages: ${statusObj.name}`)
```
Comment (anuveyatsu) [inline index.js]: "packages" should be singular as it is referring to a single package

File: index.js [hunk 3]
```diff
@@ -62,11 +56,11 @@ class CoreTools {
           console.log(`valid`)
         } else {
           error(result)
-          //const result = CSV.stringify(result)
+          // Const result = CSV.stringify(result)
```
Comment (anuveyatsu) [inline index.js]: Why don't you just delete that line?

File: index.js [hunk 4]
```diff
@@ -88,28 +82,33 @@ class CoreTools {
...
       const pkg = await Package.load(statusObj.local)
-      //push to DataHub
+      //  Push to DataHub
       await datahub.push(pkg)
       console.log(`🙌 pushed ${statusObj.name}`)
     }
   }
 
-  // TODO: save pkg statuses to csv at path
+  //  TODO: ave pkg statuses to csv at path
```
Comment (anuveyatsu) [inline index.js]: what's ave?

File: index.js [hunk 5]
```diff
@@ -88,28 +82,33 @@ class CoreTools {
...
-    fs.writeFile(path_, csv, function (err) {
-      if (err) return console.log(err);
+    const fields = ['name', 'github_url', 'runDate', 'validated', 'message', 'published']
+    const csv = json2csv({
+      data: this.statuses,
+      fields
+    })
+    fs.writeFile(path_, csv, err => {
+      if (err) {
+        return console.log(err)
```
Comment (anuveyatsu) [inline index.js]: I think you shouldn't return console.log

File: package.json [hunk 6]
```diff
@@ -18,11 +34,16 @@
   "homepage": "https://github.com/datahq/core-datasets-tools#readme",
   "dependencies": {
     "ava": "^0.21.0",
+    "babel-core": "^6.25.0",
```
Comment (anuveyatsu) [inline package.json]: why do we need babel-core?

**Review by anuveyatsu — CHANGES_REQUESTED** [review]
Please see my comments above.

Also, can you update `README` (as a separate commit):
* add `xo` badge (see how it's done in cli repo)
* update instructions on cloning, checking and pushing

File: index.js (hunk 3 above)
Comment (Mikanebu) [reply index.js]: forgot it. Very detailed review

File: index.js (hunk 4 above)
Comment (Mikanebu) [reply index.js]: it is save, corrected

File: index.js (hunk 5 above)
Comment (Mikanebu) [reply index.js]: it will be just console.log?

Comment (anuveyatsu) [reply index.js]: @Mikanebu yes

**Comment by Mikanebu** [comment]
@anuveyatsu review

**Review by anuveyatsu — APPROVED** [review]
OK

**Pull request merged**